test/page_toc_filter_test.rb

```diff
@@ -16,9 +16,9 @@ def test_toc_levels
   end
 
   def test_misaligned_toc
-    assert_raises ArgumentError do
-      pipeline(fixture('misaligned_toc.txt'), toc_levels: 'h1, h2, h3, h4')
-    end
+    #assert_raises ArgumentError do
```

Comment:
If this test is no longer needed maybe just remove it instead of commenting it out?